exo-create/main_test.go

```diff
@@ -0,0 +1,162 @@
+package main_test
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path"
+	"strings"
+	"testing"
+
+	"github.com/DATA-DOG/godog"
+	"github.com/DATA-DOG/godog/gherkin"
+	"github.com/Originate/exosphere/exo-create/test_helpers"
+)
+
+var cmd *exec.Cmd
+var in io.WriteCloser
+var out bytes.Buffer
+var appDir string
+var err error
+
+type AppConfig struct {
+	AppName, AppVersion, ExocomVersion, AppDescription string
+}
+
+func run(cwd, command string) error {
+	cmdSegments := strings.Split(path.Join(cwd, "bin", command), " ")
```

Comment:
I've changed `run` to take in a string array but I still need to process `command` in a separate function because it is passed in from cucumber tests as a single string.